src/components/Footer/component/FooterMain/FooterMain.jsx

```diff
@@ -0,0 +1,96 @@
+import React from 'react';
+import './FooterMain.scss';
+import { Link } from 'react-router-dom';
+
+const FooterMain = () => {
+  return (
+    <div className="footer-main-section">
```

Comment:
```suggestion
    <div className="footer-main">
```
컴포넌트 이름이랑 같도록 수정 해 주세요

nesting할 때 최대한 겹치지 않도록 하기 위한 wecode convention입니다.

Reply:
컴포넌트 이름과 동일하게 변경했습니다~